src/main/scala/org/phenoscape/kb/KB.scala

```diff
@@ -189,8 +190,44 @@ OPTIONAL {
     App.executeSPARQLQueryString(query.text, res => Instant.parse(res.getLiteral("date").getLexicalForm)).map(_.head)
   }
 
+  def getKBMetadata: Future[KBMetadata]= {
+    val builtFut = buildDate
+    val ontologiesFut = kbOntologies
+    for {
+      built <- builtFut
+      ontologies <- ontologiesFut
+    } yield KBMetadata(built, ontologies)
+  }
+
+  def kbOntologies: Future[Set[(IRI, IRI)]] = {
+    val query = sparql"""
+    PREFIX owl: <http://www.w3.org/2002/07/owl#>
+    SELECT ?ont ?version
+    FROM $KBMainGraph
+    WHERE {
+      ?ont $rdfType owl:Ontology .
+  	  ?ont owl:versionIRI ?version
+    }
+  """
+
+    App.executeSPARQLQueryString(query.text, qs => (IRI.create(qs.getResource("ont").getURI), IRI.create(qs.getResource("version").getURI))).map(_.toSet)
+  }
 }
 
+case class KBMetadata(built: Instant, ontologies: Set[(IRI, IRI)]) extends JSONResultItem {
+
+  def toJSON: JsObject = Map(
+    "build_time" -> built.toString.toJson,
+    "ontologies" -> ontologies.toSeq.sortBy(_.toString()).map{
```

Comment:
Remove parentheses on `toString` call (using parentheses on nullary methods is Scala convention for side-effecting call). Need a space within `map{`.

Reply:
Why do we need space after `map`? Is there some place I can read about such conventions?

Reply:
Just for consistency (I can't explain why it looks better; I think it does). The default IntelliJ formatter should do this. We could look into integrating [scalafmt](https://scalameta.org/scalafmt/) to separate this from IntelliJ.